scripts/twister

```diff
@@ -772,6 +772,12 @@ structure in the main Zephyr tree: boards/<arch>/<board_name>/""")
              "stdout detailing RAM/ROM sizes on the specified filenames. "
              "All other command line arguments ignored.")
 
+    parser.add_argument("--serial-decode-specified", action="store_true",
```

Comment:
do we really need an option for this? Can you please implement it  generically so that it does not require an option? I am not sure why you tie this to a board, this is probably related to your environment where you flash and test this board, it has nothing to do with the board.

Reply:
Hi @nashif , it is related to acrn_ehl_crb serial output, other board such as ehl_crb or up2 does not have this problem. About the environment, I'll try again to see if we can have another way to avoid this. I just thought implementing an option is a more generic way than adding things like "if self.board == ''acrn_ehl_crb" in the code. Seems like my idea is not correct. Thanks for your review!

Reply:
Wouldn't it be easier to do try/except in this case? If utf-8 or default encoding doesn't work, try latin-1, or iso-8859-1.

Reply:
Hi @dcpleung , good suggestion, will change to this, thx!